### PR TITLE
Update FAQ-Introduction.md

### PR DESCRIPTION
**Pull request by Trxlz**

### CODE REVIEW

File: docs/FAQ/FAQ-Introduction.md [hunk 1]
```diff
@@ -167,48 +166,48 @@ Key dates:
...
-Users who aren't programmers also benefit from open-source software, because they can use this software for any purpose they wish, not merely the way someone else thinks they should.
+Users who aren't programmers also benefit from open-source software, since they can use this software for any purpose.
 
 :::tip Training
 Other people like open-source software because it helps them become better programmers.
 :::
 
-Because open-source code is publicly accessible, students can easily study it as they learn to make better software.
-Students can also share their work with others, inviting comment and critique, as they develop their skills.
+Open-source code is publicly accessible.Students can easily study it as they learn to make better software.
```
Comment (RiccardoMasutti) [inline docs/FAQ/FAQ-Introduction.md:183]:
```suggestion
Open-source code is publicly accessible. Students can easily study it as they learn to make better software.
```

**Review by RiccardoMasutti — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-Introduction.md [hunk 2]
```diff
@@ -130,24 +130,23 @@ Also, remember to follow our [blog](https://blog.wasabiwallet.io) to get the lat
 
 :::details
 ### Can the coordinator attack me?
-
+The developers have gone to great lengths to ensure that the coordinator cannot steal funds nor link inputs to outputs.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Introduction.md:133]:
```suggestion

The developers have gone to great lengths to ensure that the coordinator cannot steal funds nor link inputs to outputs. 
```

File: docs/FAQ/FAQ-Introduction.md (hunk 1 above)
Comment (yahiheb) [inline docs/FAQ/FAQ-Introduction.md:183]:
```suggestion
Open-source code is publicly accessible.
Students can easily study it as they learn to make better software.
```

File: docs/FAQ/FAQ-Introduction.md [hunk 3]
```diff
@@ -167,48 +166,48 @@ Key dates:
...
+Open-source code is publicly accessible.Students can easily study it as they learn to make better software.
+Students can also share their work with others, inviting comments and critique, as they develop their skills.
 When people discover mistakes in programs' source code, they can share those mistakes with others to help them avoid making those same mistakes themselves.
 
 :::tip Security
 Some people prefer open-source software because they consider it more secure and stable than proprietary software.
 :::
 
-Because anyone can view and modify open-source software, someone might spot and correct errors or omissions that a program's original authors might have missed.
+Anyone can view and modify open-source software. Other users may spot and correct errors or omissions that a program's original authors might have missed.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Introduction.md:191]:
```suggestion
Anyone can view and modify open-source software.
Other users may spot and correct errors or omissions that a program's original authors might have missed.
```

File: docs/FAQ/FAQ-Introduction.md [hunk 4]
```diff
@@ -167,48 +166,48 @@ Key dates:
...
 And because so many programmers can work on a piece of open-source software without asking for permission from original authors, they can fix, update, and upgrade open-source software more quickly than they can proprietary software.
 
 :::tip Stability
 Many users prefer open-source software to proprietary software for important, long-term projects.
 :::
 
-Because programmers publicly distribute the source code for open-source software, users relying on that software for critical tasks can be sure their tools won't disappear or fall into disrepair if their original creators stop working on them.
+Programmers publicly distribute the source code for open-source software. Users relying on that software for critical tasks can be sure their tools won't disappear or fall into disrepair if their original creators stop working on them.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Introduction.md:198]:
```suggestion
Programmers publicly distribute the source code for open-source software.
Users relying on that software for critical tasks can be sure their tools won't disappear or fall into disrepair if their original creators stop working on them.
```

File: docs/FAQ/FAQ-Introduction.md [hunk 5]
```diff
@@ -167,48 +166,48 @@ Key dates:
...
 
 :::details
 ### What is the general idea of ZeroLink CoinJoin?
 
 While fungibility is an essential property of good money, Bitcoin has its limitations in this area.
-Numerous fungibility improvements have been proposed; however none of them have addressed the privacy issues in full.
+Numerous fungibility improvements have been proposed; however, none of them have addressed the privacy issues in full.
 ZeroLink is designed so that no participant or outside observer can spy on the user.
-The scope of ZeroLink is not limited to a single transaction, it extends to transaction chains and it addresses various network layer deanonymizations, however its scope is limited to Bitcoin's first layer.
+The scope of ZeroLink is not limited to a single transaction, it extends to transaction chains and it addresses various network layer deanonymizations. However, its scope is limited to Bitcoin's first layer.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Introduction.md:208]:
```suggestion
The scope of ZeroLink is not limited to a single transaction, it extends to transaction chains and it addresses various network layer deanonymizations.
However, its scope is limited to Bitcoin's first layer.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by yahiheb** [comment]
@Trxlz We have this writing convention that each sentence should be in a new line. https://github.com/zkSNACKs/WasabiDoc#writing-conventions

**Review by yahiheb — APPROVED** [review]

**Review by RiccardoMasutti — APPROVED** [review]

**Pull request merged**